ww3tools/SatGlobal_Altimeter_switch.py

```diff
@@ -0,0 +1,261 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+
+
+#Purpose and usage:
+
+# This Python script is designed to process satellite data from netCDF files for a specific satellite (e.g., JASON-2).
+# It can perform time averaging on the data if desired and save the results in new NetCDF files.
+#Altimeters must have been previously downloaded. Path where altimeter data is saved must be informed and
+#edited (see dirs below).
+
+
+#USAGE:
```

Comment:
Can this usage statement be updated a bit so I know how to test this?   At this time, I don't have quite enough information to test at this moment. 

Also, ideally we'll have an option to turn on/off the time-avaeraging and there's no need to have both this and the other file without the option.